src/features/remote-control-menu/buttons/wireless-screenshare-button.js

```diff
@@ -14,10 +14,14 @@ import styles from '../remote-control-menu.css';
  * @returns {ReactElement}
  */
 export default function WirelessScreenshareButton(props) {
-    const { isProxyConnectionActive, isScreensharing, onClick } = props;
+    const {
+        isScreensharing,
+        isWirelessScreenshareConnectionActive,
+        onClick
+    } = props;
     let buttonText;
 
-    if (isProxyConnectionActive && !isScreensharing) {
+    if (isWirelessScreenshareConnectionActive && !isScreensharing) {
```

Comment:
what does `isScreensharing` represent here?  If `isScreensharing` is false we show 'cancel screensharing` ?

Oh, does this test represent an state in between starting the connection and actually transmitting?